src/ext/JourneyPatternStopPointMap/Popovers/SearchResult.tsx

```diff
@@ -3,14 +3,13 @@ import { Heading5 } from '@entur/typography';
 import { PositionIcon } from '@entur/icons';
 import React from 'react';
 import { TertiarySquareButton } from '@entur/button';
-import { useMap } from 'react-leaflet';
-import { getStopPlaceLocation } from '../StopPlaceMarker';
 import {
   FocusedMarker,
   JourneyPatternMarker,
   JourneyPatternMarkerType,
 } from '../types';
 import { determineQuayToFocus } from '../helpers';
+import { getStopPlaceLocation } from '../StopPlaceMarker';
```

Comment:
This function seems to belong in a helpers / utility file we can get rid of this awkward dependency to the StopPlaceMarker file.